src/mlpack/tests/cv_test.cpp

```diff
@@ -187,6 +188,27 @@ BOOST_AUTO_TEST_CASE(R2ScoreTest)
   BOOST_REQUIRE_CLOSE(R2Score::Evaluate(lr, data, responses), expectedR2, 1e-5);
 }
 
+/**
+ * Test the topK Accuracy metric (TopKAccuracy).
+ */
+BOOST_AUTO_TEST_CASE(TopKAccuracyTest)
+{
+  // Using the same data for training and testing.
+  arma::mat data = arma::linspace<arma::rowvec>(1.0, 10.0, 10);
+  // Labels that will be considered as "ground truth".
+  arma::Row<size_t> labels("0 1  0 2  1 2 2 2  3 3 3 3");
+  // prediction probabilities for each class
+  arma::rowvec predsprob("0 0  0 1  2.005 2.005 2.005 2.005  3 3 3 3");
```

Comment:
```suggestion
  arma::Row<size_t> labels("0 1 0 2 1 2 2 2 3 3 3 3");
  // prediction probabilities for each class
  arma::rowvec predsprob("0 0 0 1 2.005 2.005 2.005 2.005 3 3 3 3");
```
Spacing Fix. :)
Thanks.